rest/data/aliases.go

```diff
@@ -127,7 +127,11 @@ func updateAliasesForSection(projectId string, updatedAliases []restModel.APIPro
 // validateFeaturesHaveAliases returns an error if project/repo aliases are not defined for a Github/CQ feature.
 // Does not error if version control is enabled.
 func validateFeaturesHaveAliases(pRef *model.ProjectRef, aliases []restModel.APIProjectAlias) error {
-	if pRef.IsVersionControlEnabled() {
+	fullProjectRef, err := model.FindBranchProjectRef(pRef.Id)
```

Comment:
This doesn't take into account the repo ref so this isn't quite right. I think this should instead take in the original project ref as well as the new project ref (which is what it currently takes in) which functions calling it should have available (example: https://github.com/evergreen-ci/evergreen/blob/358b4af89e357a1db8d44b91603b3817e6796e73/rest/data/project_settings.go#L118)

Reply:
Sounds good, did this and tested again locally to confirm